tmail-backend/apps/distributed/src/main/java/com/linagora/tmail/james/app/DistributedServer.java

```diff
@@ -312,7 +313,8 @@ protected void configure() {
             new TeamMailboxModule(),
             new TMailMailboxSortOrderProviderModule(),
             new TmailEventModule(),
-            new TmailEventDeadLettersModule());
+            new TmailEventDeadLettersModule(),
+            new OpenPaasModule());
```

Comment:
```
22:23:34.258 [ERROR] c.l.t.OpenPaasModule - Could not find configuration file 'openpaas.properties'
22:23:34.271 [ERROR] o.a.j.GuiceJamesServer - Fatal error while starting James
java.io.FileNotFoundException: openpaas
```

Likely we should make the OpenPaaS module optional?

Reply:
Fully agree.

If unspecified we shall fallback to the main rabbitMQ connection and rely on / vhost.